RevenueCatUI/CustomerCenter/Data/PurchaseInformation.swift

```diff
@@ -115,7 +135,7 @@ struct PurchaseInformation {
         // Use entitlement data if available, otherwise derive from transaction
         if let entitlement = entitlement {
             self.explanation = entitlement.explanation
-            self.expirationOrRenewal = entitlement.expirationOrRenewal(dateFormatter: dateFormatter)
+            self.expirationOrRenewal = entitlement.expirationOrRenewal(dateFormatter: Self.defaultDateFormatter)
```

Comment:
Shouldn't this use the `dateFormatter` parameter?

```suggestion
            self.expirationOrRenewal = entitlement.expirationOrRenewal(dateFormatter: dateFormatter)
```

Reply:
I've applied it https://github.com/RevenueCat/purchases-ios/pull/5086